dexios/src/global/states.rs

```diff
@@ -40,19 +40,19 @@ pub enum HeaderLocation {
     Detached(String),
 }
 
-#[derive(PartialEq, Clone, Copy)]
+#[derive(PartialEq, Eq, Clone, Copy)]
 pub enum EraseMode {
     EraseFile(i32),
     IgnoreFile,
 }
 
-#[derive(PartialEq, Clone, Copy)]
+#[derive(PartialEq, Eq, Clone, Copy)]
 pub enum HashMode {
     CalculateHash,
     NoHash,
 }
 
-#[derive(PartialEq, Copy, Clone)]
+#[derive(PartialEq, Eq, Copy, Clone)]
```

Comment:
I'm a little unsure why we're deriving `Eq` now also? @pleshevskiy

Reply:
I don't know) Didn't you change the clippy settings?)

Reply:
I didn't - I use `cargo clippy --all -- -W clippy::pedantic -W clippy::correctness -W clippy::perf -W clippy::style -W clippy::suspicious -W clippy::complexity -D warnings` for everything :)

Reply:
:thinking: it's very strange) because I use project clippy settings